docs/starters.yml

```
@@ -6300,6 +6300,38 @@
     - Twitter Cards meta
     - Sitemap Generation
     - XML Sitemaps
+    - Progressive Web Ap
+- url: https://a2zarslaan.github.io/gatsby-starter-sasslan/
+  repo: https://github.com/a2zarslaan/gatsby-starter-sasslan
+  description: A minimalistic Gatsby starter template featuring SASS and CSS 7-1 architecture.
+  tags:
+    - Blog
+    - Portfolio
+    - Markdown
+    - CSS Modules
+    - React Helmet
+    - SASS
+    - Page Transitions
+    - SVG
+    - Google Fonts
+    - Responsive Design
+    - Github Pages
+    - Landing Page
+    - Beautiful typography
```

```suggestion
    - Blog
    - Portfolio
    - Markdown
```

These tags are supported in our whitelist